File: system-test/sec-tests/java/Sec57AuditLog.java
// SEC-57: Audit Log Missing
// Test: TC-CR-060
package com.example;

import java.sql.*;

public class AdminService {

    public void deleteUser(int userId) throws Exception {
        Connection conn = getConnection();
        Statement stmt = conn.createStatement();
        stmt.execute("DELETE FROM users WHERE id = " + userId);
        // No audit log of who deleted this user or when
    }
}

File: system-test/sec-tests/python/sec_57_audit_log.py
# SEC-57: Audit Log Missing
# Test: TC-CR-060
import sqlite3

def delete_user(user_id):
    conn = sqlite3.connect('app.db')
    conn.execute("DELETE FROM users WHERE id = ?", (user_id,))
    conn.commit()
    # No audit log of who deleted this user or when
